### PR TITLE
fixed text overflow

### PR DESCRIPTION
**Pull request by KcPele**

### CODE REVIEW

**Comment by sverps** [comment]
You forgot to link back to the issue, always helpful to quickly find what a PR is fixing :)

> Fixes #205

**Comment by carletex** [comment]
Also adding screenshots (before / after) is useful!

**Review by carletex — CHANGES_REQUESTED** [review]
Left a comment @KcPele 

Thanks

File: packages/nextjs/components/scaffold-eth/Contract/ContractUI.tsx [hunk 1]
```diff
@@ -86,7 +86,7 @@ const ContractUI = ({ contractName }: TContractUIProps) => {
               </p>
             )}
           </div>
-          <div className="bg-base-300 rounded-3xl px-6 lg:px-8 py-4 shadow-lg shadow-base-300">
+          <div className="bg-base-300 rounded-3xl break-all px-6 lg:px-8 py-4 shadow-lg shadow-base-300">
```
[the remark this answers is not in the document]
Comment (KcPele) [reply packages/nextjs/components/scaffold-eth/Contract/ContractUI.tsx:95]: done. i have made the changes

[the remark this answers is not in the document]
Comment (carletex) [reply packages/nextjs/components/scaffold-eth/Contract/ContractUI.tsx:95]: Since you added it on `DisplayVariables.tsx` you can remove it from here

**Review by carletex — CHANGES_REQUESTED** [review]
By mistake, you also added this to the commits. 

image

Can you remove it?

**Comment by KcPele** [comment]
> By mistake, you also added this to the commits.
> 
> image
> 
> Can you remove it?

please how do I remove it? it was a mistake.

**Comment by technophile-04** [comment]
> please how do I remove it? it was a mistake.
should I revert to the previous commit and make the changes again then push?

Deleted at [532172a](https://github.com/scaffold-eth/se-2/pull/207/commits/532172a3508e4326540638faf22cfcc7bcd8ee77), I think you might have faced issue while deleting because of `.git cache` 

checkout -> https://stackoverflow.com/questions/1260748/how-do-i-remove-a-submodule

Also synced with `main` branch at [50e02b7](https://github.com/scaffold-eth/se-2/pull/207/commits/50e02b712def8df9936d9c8e6881547639d841a2)

Thanke @KcPele 🙌

File: packages/nextjs/components/scaffold-eth/Contract/DisplayVariables.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const DisplayVariable = ({ contractAddress, functionFragment, refreshDisplayVari
   return (
     <div className="space-y-1 pb-2">
       <div className="flex items-center gap-2">
-        <h3 className="font-medium text-lg mb-0 break-words">{functionFragment.name}</h3>
+        <h3 className="font-medium text-lg mb-0 break-all">{functionFragment.name}</h3>
```
Comment (carletex) [inline packages/nextjs/components/scaffold-eth/Contract/DisplayVariables.tsx:41]: This is the h3. We wanted it on the result div

**Pull request merged**